crates/rpc/rpc/src/eth/core.rs

```diff
@@ -428,6 +436,20 @@ where
     pub const fn blocking_task_guard(&self) -> &BlockingTaskGuard {
         &self.blocking_task_guard
     }
+
+    /// Returns [`broadcast::Receiver`] of new raw transactions
+    #[inline]
+    pub fn subscribe_to_raw_transactions(&self) -> broadcast::Receiver<Bytes> {
+        self.raw_tx_sender.subscribe()
+    }
+
+    /// Broadcasts raw transaction if there are active subscribers.
+    #[inline]
+    pub fn broadcast_raw_transaction(&self, raw_tx: Bytes) {
+        if self.raw_tx_sender.receiver_count() > 0 {
+            let _ = self.raw_tx_sender.send(raw_tx);
+        }
```

Comment:
like this, this check would be redundant because at this time we have already cloned the value and this is again checked on send.